streams/src/main/java/org/apache/kafka/streams/processor/internals/UnwindowedChangelogTopicConfig.java

```diff
@@ -25,7 +25,7 @@
 
 /**
  * UnwindowedChangelogTopicConfig captures the properties required for configuring
- * the un-windowed store changelog topics.
+ * the un-windowed, un-versioned store changelog topics.
```

Comment:
This class name `UnwindowedChangelogTopicConfig` is a bit outdated now and should really be `UnwindowedUnversionedChangelogTopicConfig`. A bit clunky but I can't think of anything better. Curious if reviewer's think it's worth the rename.